web/src/main/java/org/springframework/security/web/DefaultSecurityFilterChain.java

```diff
@@ -61,4 +63,5 @@ public boolean matches(HttpServletRequest request) {
 	public String toString() {
 		return "[ " + requestMatcher + ", " + filters + "]";
 	}
+
```

Comment:
let's use the existing formatter so this does not occur

Reply:
What do you mean by existing formatter??

Reply:
I meant, these whitespaces means that the code formatter we have used for this PR does not match the one used originally. Formatter can also mean any settings in your IDE that indents/formats/structures code. Please see the section on whitespaces in contributing guidelines